pages/clustering/high-availability.mdx

```diff
@@ -267,21 +270,13 @@ This operation will result in writing to the Raft log.
 You can check the state of the whole cluster using the `SHOW INSTANCES` query. The query will show which instances are visible in the cluster,
 which network ports they are using for managing cluster state, whether they are considered alive from the coordinator's
 perspective, their role (are they MAIN, REPLICA, LEADER, FOLLOWER or unknown if not alive) and the time elapsed in ms since the last successful response
-from instances to the leader's ping. This query can be run both on leader and followers with the sole difference that followers cannot know the 
-health status of other instances and the time of last response since only the leader is pinging instances for their status.
+from instances to the leader's ping. This query can be run both on leader and followers. Followers will try to contact leader to get state of the cluster. 
+In case contacting leader fails, followers will return data with time elapsed in ms since the last successful response set to 0, and health state unknown.
```

Comment:
s/contacting leader/contacting the leader
s/with time/with the time

And we need to better indicate that the phrase "time elapsed in ms since the last successful response" is one specific thing. It's a long phrase and hard to parse.

Reply:
Agreed, changed phrase to `response time`